vms/platformvm/state/state.go

```diff
@@ -229,11 +235,13 @@ type state struct {
 
 	currentHeight uint64
 
-	addedBlocks map[ids.ID]stateBlk // map of blockID -> Block
-	// cache of blockID -> Block
-	// If the block isn't known, nil is cached.
-	blockCache cache.Cacher[ids.ID, *stateBlk]
-	blockDB    database.Database
+	addedBlockIDs map[uint64]ids.ID            // map of height -> blockID
```

Comment:
Should we name this `addedBlockHeights`, sounds more consistent with other fields (blocks maps block IDs to ..., block heights maps block heights to ...)

Reply:
@dhrubabasu @StephenButtolph Also, do you see a future we may support multiple heights or ranges for "get blocks by heights" API? If we ever support returning multiple block ids within a range, we might as well use btree here for fast range lookups.

Reply:
This is the same naming as the X-chain

I think a range lookup is outside the scope of this PR even if we were considering adding it

Reply:
I don't think using a B-tree would actually improve the performance of a range query here (as we can just iterate over the range)